sdk/python/kfp/components_tests/test_data_passing.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from ..components import _data_passing
```

Comment:
from kfp.components ...

Reply:
Unfortunately, our presubmit test setup only copies the sub-folders:
https://github.com/kubeflow/pipelines/blob/706c105c8ca3977a91b54d74caebe8ffb39f6aa7/test/presubmit-tests-sdk.sh#L28

and `kfp/components` isn't set up as independent [namespace package](https://packaging.python.org/guides/packaging-namespace-packages/), so `kfp` wouldn't be recognized here.

As a result, all unit tests under this folder have to use relative imports.

Ideally, we should fix the whole file structures for v1, and makes test files sit next to their source files like 
- data_passing.py
- data_passing_test.py

Reply:
Ack, thanks for explaining! Let's fix it at some later point down the road.